components/landing/TheQiskitCapabilitiesSection/index.vue

```diff
@@ -0,0 +1,22 @@
+<template>
+  <article class="bx--grid page-section qiskit-capabilities-section">
+    <h2 v-text="title" />
+  </article>
+</template>
+
+<script lang="ts">
+import Vue from 'vue'
+import { Component, Prop } from 'vue-property-decorator'
```

Comment:
If we remove the props (comments in lines 13 and 14):

```suggestion
import { Component } from 'vue-property-decorator'
```